packages/near-membrane-base/src/membrane.ts

```diff
@@ -611,6 +644,42 @@ export function createMembraneMarshall() {
             return fn;
         }
 
+        function isLiveObject(value: any): boolean {
+            if (value === null || value === undefined) {
+                return false;
+            }
+            if (typeof value === 'object') {
+                if (
+                    // Arrays are marked as live in the BoundaryProxyHandler constructor.
+                    // isArrayOrNotOrThrowForRevoked(value) ||
+                    ArrayBufferIsView(value)
+                ) {
+                    return true;
+                }
+                // Plain object detection based on ObservableMembrane
+                // https://github.com/salesforce/observable-membrane/blob/v2.0.0/src/observable-membrane.ts#L46-L47
+                const proto = ReflectGetPrototypeOf(value);
+                if (
+                    proto === ObjectProto ||
+                    proto === null ||
+                    ReflectGetPrototypeOf(proto) === null
+                ) {
+                    return true;
+                }
+            }
+            if (
+                !ReflectApply(ObjectProtoHasOwnProperty, value, [
+                    LOCKER_LIVE_INSTANCE_MARKER_SYMBOL,
```

Comment:
this apply is done twice in this method, it is always false at this point.

Reply:
The `ObjectProtoHasOwnProperty` checks are for two different properties. The follow-up check for `LOCKER_LIVE_INSTANCE_MARKER_SYMBOL` is to make sure it isn't an own property but rather inherited.